### ISSUE TITLE
Cart page - Consultant Photo is not displayed

### ISSUE THREAD

**Issue by suehamre** [tastefully-simple/cornerstone#293]
### Expected behavior
The tan banner in the cart page should display the consultants photo if they have one uploaded. If they do not have a photo updated, it should display the teal lady photo. 
Mock up: 
![image](https://user-images.githubusercontent.com/57115316/91100020-02e81300-e62a-11ea-80d9-45dcb0e05307.png)
If no photo uploaded cart page would display this: 
![image](https://user-images.githubusercontent.com/57115316/91100165-4e022600-e62a-11ea-9c80-3b4eab4a0d0a.png)

### Actual behavior
The cart page is displaying a food photo. 
![image](https://user-images.githubusercontent.com/57115316/91099966-e8159e80-e629-11ea-8a3d-5f6f8a710f09.png)

### Steps to reproduce behavior
Selected a consultant that has a photo uploaded and added items to the cart. View the cart page. 
![image](https://user-images.githubusercontent.com/57115316/91100093-2d39d080-e62a-11ea-8d26-882f5b0d94b3.png)

**Comment by jsmnglng-w** [comment]
Created a ticket for this issue: [TST-180](https://razoyo.atlassian.net/browse/TST-180)

**Comment by wbyrnetx** [comment]
This has passed our QA and is ready for you to test.

**Comment by suehamre** [comment]
8-26 SH - Fixed. :)

**Comment by suehamre** [comment]
9-4 Somehow this got broke again. 
![image](https://user-images.githubusercontent.com/57115316/92274546-12315100-eeb3-11ea-8990-ccc8f529244c.png)

**Comment by jsmnglng-w** [comment]
@suehamre sorry, can you tell me what was broken?

**Comment by suehamre** [comment]
@jsmnglng-w  - The consultants photo is not displaying in the shopping cart if they have one uploaded. The example above is consultant CID#28 Betty Evans, she has a photo uploaded and in the shopping cart the generic one is displayed.

**Comment by jsmnglng-w** [comment]
@suehamre I found that it's because when searching for a consultant, the CID#28 Betty Evans displays the noconsultant image. And that is being displayed because it cannot find the consultant image so it's displaying the fallback image. Looking what the API response is, the url for CID#28 is https://tso.tastefullysimple.com/consultant/photo/bevans3.jpg 
![Screen Shot 2020-09-04 at 2 48 51 PM](https://user-images.githubusercontent.com/69249028/92280949-30517e00-eec0-11ea-9816-2994878550c1.png)

and it's throwing 404
![Screen Shot 2020-09-04 at 2 30 36 PM](https://user-images.githubusercontent.com/69249028/92280960-35163200-eec0-11ea-88ca-fbfdd22912d7.png)

**Comment by suehamre** [comment]
@jsmnglng-w , thanks for the update. FYI - the photo does appear when going to the consultants webpage. 
![image](https://user-images.githubusercontent.com/57115316/92282948-58db7700-eec4-11ea-8088-c4cc2ed688a2.png)

@TastefullySimple  - Jared, is the above URL trying to go to sitecore TSO? 
![image](https://user-images.githubusercontent.com/57115316/92282983-714b9180-eec4-11ea-8c79-dc44139df548.png)

**Comment by suehamre** [comment]
9-9 I am now seeing the consultants photo displayed in the cart again. I'm not sure what's changed. This morning I seen it in the cart but not on the webpage. I went back in again and was on the webpage, the cart and the search results. Seems to be something with a connection somewhere maybe because it's not consistent. Razoyo is seeing the photos as well, I had Monica try and she seen some blips but they appeared. Closing bug.
